bin/get_metadata.py

```diff
@@ -82,6 +83,14 @@ def get_bcl2fastq_version(self, runfolder):
             bcl2fastq_str = f.read()
         return bcl2fastq_str.split("v")[1].strip()
 
+    def get_software_version(self, runfolder):
+        with open(Path(runfolder) / "pipeline_info" / "software_versions.yml") as f:
+            return dict(
+                line.strip().split(": ")
+                for line in f.readlines()
+                if line.startswith("  ")
```

Comment:
Ah smart! But I wonder if this could be dangerous if the yaml is formatted in a different way? i.e the indent is another number of spaces. I guess this is a nf-core standard so the risk is probably small, but still. Could potentially be safer to import it with a yaml package and handle the values of all top-level keys. Let me know what you think.

Reply:
Good point, fixed it now :+1: